samples/TypeScript/js-require.tsx

```diff
@@ -0,0 +1 @@
+const React = require("react")
```

Comment:
We could also just go with the common cases, rather than all the variants that could exist?